### ISSUE TITLE
Instruction on how to run IP audit locally needed 

### ISSUE THREAD

**Issue by Leectan** [SecurityFTW/cs-suite#9]
I believe a quickstart or quick instruction on how to run IP audit locally (Mac, Windows, Linux) or remote server (Ec2, Google Compute Engine, Azure) would be great.

**Comment by shivankar-madaan** [comment]
So we have parameters option as -aip (audit_ip) which is manadatory
-u user-name,if you want to use any other user than ec2-user(which is default)
-pem pem file location
-p password,this is basically if you use password along with the pem file,or if you use just the password and not the pem file
So some the examples can be like
`python cs.py -aip 10.2.2.3 -pem test.pem`
`python cs.py -aip 10.2.2.3 -u shiv -pem test.pem`
`python cs.py -aip 10.2.2.3 -u shiv -p` (you would get a password prompt)

So basically depends on how you connect to your machine usually,and in the same way you can pass the same to cs.py

**Comment by shivankar-madaan** [comment]
Also currently the local Audit is just for AWS infra
We maybe be adding GCP in the future

**Comment by shivankar-madaan** [comment]
@Leectan were you able to run the local audit?
